server.js

```diff
@@ -69,7 +67,7 @@ class Metadata {
         }]);
         self._p.on('data', () => {});
         self._p.on('end', () => {
-            if (!args.redis) {
+            if (INDEXER !== "redis") {
                 this.idx.build();
```

Comment:
why not have a build method in all indexers and just call that - the lunr build method could just be a no-op. one should always try to avoid if-then statements when doing OOP - thats what inherritance is for :-)

Reply:
I have added the build method to the redisIndexer.